colossalai/inference/tensor_parallel/engine.py

```diff
@@ -181,10 +181,11 @@ def prepare_batch_state(self, inputs) -> BatchInferState:
         max_len_in_batch = -1
         if isinstance(inputs, (BatchEncoding, dict)):
             for i, attn_mask in enumerate(attention_mask):
-                if isinstance(attn_mask, torch.Tensor):
-                    curr_seq_len = int(torch.sum(attn_mask))
-                else:
-                    curr_seq_len = int(sum(attn_mask))
+                curr_seq_len = len(attn_mask)
```

Comment:
This is a temporary fix. In future we might revise it by concatenating input ids into a single tensor and modify model modelling (e.g. embedding in model forward, decoder layer, etc)